[2023-06-19 20:26:43,676] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dag_weather.extract_data manual__2023-06-19T20:16:30.544131+00:00 [queued]>
[2023-06-19 20:26:43,688] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dag_weather.extract_data manual__2023-06-19T20:16:30.544131+00:00 [queued]>
[2023-06-19 20:26:43,688] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-06-19 20:26:43,688] {taskinstance.py:1377} INFO - Starting attempt 3 of 6
[2023-06-19 20:26:43,688] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-06-19 20:26:43,707] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): extract_data> on 2023-06-19 20:16:30.544131+00:00
[2023-06-19 20:26:43,713] {standard_task_runner.py:52} INFO - Started process 253 to run task
[2023-06-19 20:26:43,716] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag_weather', 'extract_data', 'manual__2023-06-19T20:16:30.544131+00:00', '--job-id', '68', '--raw', '--subdir', 'DAGS_FOLDER/dag_weather.py', '--cfg-path', '/tmp/tmpnlmlsuil', '--error-file', '/tmp/tmp6cdk_pgg']
[2023-06-19 20:26:43,718] {standard_task_runner.py:80} INFO - Job 68: Subtask extract_data
[2023-06-19 20:26:43,798] {task_command.py:371} INFO - Running <TaskInstance: dag_weather.extract_data manual__2023-06-19T20:16:30.544131+00:00 [running]> on host 871cafd9693a
[2023-06-19 20:26:43,890] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=MariaRO
AIRFLOW_CTX_DAG_ID=dag_weather
AIRFLOW_CTX_TASK_ID=extract_data
AIRFLOW_CTX_EXECUTION_DATE=2023-06-19T20:16:30.544131+00:00
AIRFLOW_CTX_TRY_NUMBER=3
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-06-19T20:16:30.544131+00:00
[2023-06-19 20:26:43,891] {logging_mixin.py:115} INFO - ***************
[2023-06-19 20:26:43,891] {logging_mixin.py:115} INFO - 2023-06-19
[2023-06-19 20:26:54,270] {python.py:173} INFO - Done. Returned value was: None
[2023-06-19 20:26:54,284] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=dag_weather, task_id=extract_data, execution_date=20230619T201630, start_date=20230619T202643, end_date=20230619T202654
[2023-06-19 20:26:54,325] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-06-19 20:26:54,379] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
